CONTRIBUTING.md

```
@@ -0,0 +1,71 @@
+
+## Request a Feature
+
+If the project doesn't do something you need or want it to do:
+
+* Open an Issue at https://github.com/canopas/group-track-android/issues
+* Provide as much context as you can about what you're running into.
+* Please try and be clear about why existing features and alternatives would not work for you.
+
+Once it's filed:
+
+* The project team will [label the issue](#label-issues).
```

Markdownlint (0.37.0)

Link fragments should be valid

(MD051, link-fragments)